### DISCUSSION TITLE
Polishing English Docs

### DISCUSSION THREAD

**Discussion by Rage47** [PaddlePaddle/PaddleOCR#13442] [Q&A]
Just saw in the Readme smith about polishing the English text. 
Who do I contact best about this? (Experience as Linguist for about 10 years)

**Comment by GreatV** [comment]
hi @Rage47, You can just submit the relevant PR directly.

**Reply by Rage47** [reply]
Felt wrong to send PR's with somewhat code unrelated content. Anything I should look specifically for and do/ don't @GreatV?

**Reply by GreatV** [reply]
Feel free to submit PR on anything, there is nothing to worry about and we will review the final content.
